components/02-elements/button/_button.scss

```diff
@@ -44,6 +45,7 @@ $button-icon-hover-color          : $color-primary !default;
     margin: $button-margin;
     padding: $button-padding;
     border: $button-border;
+    border-radius: $button-border-radius;
     outline: $button-outline;
     background: $button-background;
     font-family: $button-font-family;
```

Comment:
line 116:
Where we use `.button__content` element? I see the styles, not an element in html...